style.css

```diff
@@ -0,0 +1,243 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-size: 62.5%;
+	background: #393634;
+}
+
+.container {
+	display: flex;
+	color: white;
+	font-size: 1rem;
+	font-family: 'Poppins', sans-serif;
+}
+
+p {
+	line-height: 1.5rem;
+	padding: 1%;
```

Comment:
do all p tags really need 1% padding? seems a bit too general and might bite you later